mars/scheduler/graph.py

```diff
@@ -813,35 +814,42 @@ def build_tensor_merge_graph(self, tensor_key):
 
         return serialize_graph(graph)
 
+    def build_fetch_graph(self, tensor_key):
+        """
+        Convert single tensor to graph which contains one tensor node and multiple chunk nodes
+        :param tensor_key: the key of tensor
+        """
+        tiled_tensor = self._get_tensor_by_key(tensor_key)
+        graph = DAG()
+        for c in tiled_tensor.chunks:
+            fetch_op = TensorFetch(dtype=c.dtype)
+            fetch_chunk = fetch_op.new_chunk(None, c.shape, c.index, _key=c.key).data
+            graph.add_node(fetch_chunk)
```

Comment:
Wait for #191 finished, don't need to add the chunk any more, just add the tensor is fine.